QR_extraction_service.ipynb

```diff
@@ -74,12 +74,23 @@
       "cell_type": "code",
       "source": [
         "import json\n",
-        "from decimal import Decimal\n",
         "from fhir.resources.observation import Observation\n",
         "from fhir.resources.codeableconcept import CodeableConcept\n",
         "from fhir.resources.coding import Coding\n",
         "\n",
-        "def create_observation(link_id, text, answer):\n",
+        "def extract_units(questionnaire_file):\n",
+        "    units_map = {}\n",
+        "    with open(questionnaire_file, 'r') as f:\n",
+        "        questionnaire = json.load(f)\n",
+        "        for item in questionnaire['item']:\n",
+        "            link_id = item['linkId']\n",
+        "            if 'extension' in item:\n",
+        "                for extension in item['extension']:\n",
+        "                    if 'url' in extension and extension['url'] == 'http://hl7.org/fhir/StructureDefinition/questionnaire-unit':\n",
```

Comment:
@NRBaconPharmD come to think of it, it probably is better to do some sort of crosswalk using the display here because it looks like [in_i] is the code and the unit would be like "in" (see [example of a height observation](https://build.fhir.org/observation-example-body-height.json.html
)). Lower priority though I'd say vs. some other changes. I can help write this function too.